package exercises

// Напиши функцию safeFunction(f func()), которая запускает переданную функцию и перехватывает возможную panic,
// выводя "Ошибка: <текст ошибки>".
func safeFunction(f func()) {
	// TODO: выполнить функцию f
}
